mysql/infoschema.go

```diff
@@ -416,8 +416,22 @@ func getIndexes(conv *internal.Conv, db *sql.DB, table schemaAndName) ([]schema.
 	}
 	return indexes, nil
 }
+
+// check dataType is spatial datatype or not.
+func checkIsSpatial(dataType string) bool {
+	for _, t := range MysqlSpatialDataTypes {
+		if t == dataType {
+			return true
+		}
+	}
+	return false
+}
+
 func toType(dataType string, columnType string, charLen sql.NullInt64, numericPrecision, numericScale sql.NullInt64) schema.Type {
 	switch {
+	// convert spatial data type into the text
```

Comment:
Can we have this case at last? If the type is not among those basic then only we want to check it for that spatial type.

Reply:
done.